test.py

```diff
@@ -563,16 +563,65 @@ def run_test(framework, cluster, repo_root):
         logger.info(msg, runtests_script, framework.name)
         raise CommandFailure(cmd_args)
 
+def setup_junit_xml():
+    """Try to download and import junit_xml"""
+    workdir = get_work_dir()
+    lib_dir = os.path.join(workdir, 'lib', 'python')
+    pip_cmd = ['pip', 'install', 'junit-xml', '-t', lib_dir]
+    subprocess.check_call(pip_cmd)
+    sys.path.append(lib_dir)
+    import junit_xml
+
+def emit_junit_xml(cluster_launch_attempts, frameworks):
+    """Write out all the test actions failures to a junit file for jenkins or
+    similar"""
+    import junit_xml
+    launch_fake_testcases = []
+    for launch_attempt in cluster_launch_attempts:
+        attempt_duration = launch_attempt.end_time - launch_attempt.start_time
+        fake_test = junit_xml.TestCase(launch_attempt.name,
```

Comment:
what i meant is, it's not a test, but junit_xml only can describe tests, so I'm pretending "create a cluster" is a test.